python/lsst/meas/modelfit/integrals.cc

```diff
@@ -21,21 +21,23 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 
+#include "pybind11/pybind11.h"
 
-%include "lsst/meas/modelfit/optimizer.i"
+#include "lsst/meas/modelfit/integrals.h"
 
-%{
-#include "lsst/meas/modelfit/GeneralPsfFitter.h"
-#include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
-%}
+namespace py = pybind11;
+using namespace pybind11::literals;
 
-%include "lsst/meas/modelfit/GeneralPsfFitter.h"
-%include "lsst/meas/modelfit/DoubleShapeletPsfApprox.h"
+namespace lsst {
+namespace meas {
+namespace modelfit {
+namespace {
 
-%pythoncode %{
-import lsst.pex.config
+PYBIND11_PLUGIN(integrals) {
+    py::module mod("integrals");
+    mod.def("phid", &detail::phid);
+    mod.def("bvnu", &detail::bvnu);
```

Comment:
This seems to define `lsst.meas.multifit.integrals.bvnu` yet when you reference it in tests/testIntegrals.py you reference it as `lsst.meas.multifit.integrals.detail.bvnu`. Could you please explain what's going on?